src/families/polkadot/NominateFlow/01-Validators.js

```diff
@@ -0,0 +1,448 @@
+    () =>
+      drawerValidator
+        ? getDrawerInfo({
+            t,
+            account,
+            onOpenExplorer,
+            maxNominatorRewardedPerValidator,
+            validator: drawerValidator,
+          })
+        : [],
+    [
+      drawerValidator,
+      t,
+      account,
+      maxNominatorRewardedPerValidator,
+      onOpenExplorer,
+    ],
+  );
+
+  const renderItem = useCallback(
+    ({ item }) => {
+      const selected = validators.indexOf(item.address) > -1;
+
+      const disabled = validators.length >= MAX_NOMINATIONS;
+
+      return (
+        <ValidatorItem
+          item={item}
+          disabled={disabled}
+          selected={selected}
+          onSelect={onSelect}
+          onClick={onOpenDrawer}
+        />
+      );
+    },
+    [validators, onSelect, onOpenDrawer],
+  );
+
+  const error = getStatusError(status, "errors");
+  const warning = getStatusError(status, "warnings");
+  const maxSelected = validators.length === MAX_NOMINATIONS;
+  const maybeChill = error instanceof PolkadotValidatorsRequired;
+  const ignoreError =
+    error instanceof PolkadotValidatorsRequired && !nominations.length; // Do not show error on first nominate
+
+  return (
+    <SafeAreaView style={[styles.root, { backgroundColor: colors.background }]}>
+      <NominationDrawer
+        isOpen={drawerInfo && drawerInfo.length > 0}
+        onClose={onCloseDrawer}
+        account={account}
+        ValidatorImage={({ size }) =>
+          drawerValidator ? (
+            <PolkadotIdenticon address={drawerValidator.address} size={size} />
+          ) : null
+        }
+        data={drawerInfo}
+      />
+      {nonValidators.length ? (
+        <WarningBox>
+          <Trans
+            i18nKey="polkadot.nominate.steps.validators.notValidatorsRemoved"
+            values={{ count: nonValidators.length }}
+          />
+        </WarningBox>
+      ) : null}
+      <SelectValidatorSearchBox
+        searchQuery={searchQuery}
+        setSearchQuery={setSearchQuery}
+      />
+      {sections.length <= 0 && (
+        <View style={styles.noResult}>
+          <LText style={styles.textCenter}>
+            <Trans
+              i18nKey="polkadot.nominate.steps.validators.noResultsFound"
+              values={{ search: searchQuery }}
+            >
+              <LText bold>{""}</LText>
+            </Trans>
+          </LText>
+        </View>
+      )}
+      <SectionList
+        sections={sections}
+        keyExtractor={(item, index) => item.address + index}
+        renderItem={renderItem}
+        stickySectionHeadersEnabled
+        renderSectionHeader={({ section: { title } }) => (
+          <LText
+            style={[styles.header, { backgroundColor: colors.lightFog }]}
+            color="grey"
+          >
+            {title}
+          </LText>
+        )}
+      />
+
+      <View
+        style={[
+          styles.footer,
+          { borderTopColor: colors.lightFog, backgroundColor: colors.white },
```

Comment:
change this to colors.card or colors.background :+1:

Reply:
Done :+1: